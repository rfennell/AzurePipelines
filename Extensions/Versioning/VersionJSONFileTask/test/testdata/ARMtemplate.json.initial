{
    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
    "contentVersion": "",
    "parameters": {  },
    "variables": {  },
    "resources": [  ],
    "outputs": {  }
}