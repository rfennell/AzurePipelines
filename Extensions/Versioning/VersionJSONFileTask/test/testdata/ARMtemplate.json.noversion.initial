{
    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
    "parameters": {  },
    "variables": {  },
    "resources": [  ],
    "outputs": {  }
}