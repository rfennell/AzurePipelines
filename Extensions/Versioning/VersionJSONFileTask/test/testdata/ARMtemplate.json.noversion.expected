{
    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
    "parameters": {  },
    "variables": {  },
    "resources": [  ],
    "outputs": {  },
"contentVersion": "1.2.3.4"
}